apps/deploy-web/src/services/wallet-balances/wallet-balances.service.ts

```diff
@@ -12,17 +12,16 @@ import { deploymentToDto } from "@src/utils/deploymentDetailUtils";
 export class WalletBalancesService {
   constructor(
     private readonly authzHttpService: AuthzHttpService,
-    private readonly axios: AxiosInstance,
-    private readonly masterWalletAddress: string,
-    private readonly apiEndpoint: string
+    private readonly chainApiHttpClient: AxiosInstance,
+    private readonly masterWalletAddress: string
   ) {}
 
   async getBalances(address: string): Promise<Balances> {
     const usdcIbcDenom = getUsdcDenom();
     const [balanceResponse, deploymentGrant, activeDeploymentsResponse] = await Promise.all([
-      this.axios.get<RestApiBalancesResponseType>(ApiUrlService.balance(this.apiEndpoint, address)),
+      this.chainApiHttpClient.get<RestApiBalancesResponseType>(ApiUrlService.balance("", address)),
```

Comment:
suggestion(non-blocking): since the base url is in the http client we can remove this param, same for the others

Reply:
yes, I just din't want to change it in this PR. Potentially we will replace this with chain sdk calls. Having a separate client, now it will be much easier to identify chain requests.

Because ApiUrlService mixes concerns right now, some urls are console-api based and others use chain api urls